Common/src/main/java/dk/gruppeseks/bodtrd/common/data/entityelements/Path.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package dk.gruppeseks.bodtrd.common.data.entityelements;
+
+/**
+ *
+ * @author Frederik
+ */
+public class Path
```

Comment:
this class looks a lot like a Stack<Position>, is there any reason you dont just inherit from Stack<Position>?